src/main.go

```diff
@@ -116,6 +119,20 @@ func (r *RadiusCredentials) StartGCWorker() {
 	go r.gcworker()
 }
 
+func (r *RadiusCredentials) accountingCron() *cron.Cron {
+        // hourly accounting cron job
+     	c := cron.New()
+	c.AddFunc("0 0 * * * *", func() {
```

Comment:
Is this really "hourly"?

Reply:
Go cron uses 6 space-separated fields, with second filed. Use `@hourly` if you prefer.